docs/serving/autoscaling/autoscaling-metrics.md

```diff
@@ -32,7 +32,9 @@ For more information about KPA and HPA, see the documentation on [Supported Auto
         metadata:
           annotations:
             autoscaling.knative.dev/metric: "concurrency"
+            autoscaling.knative.dev/target-utilization-percentage: "70" 
     ```
+autoscaling.knative.dev/target-utilization-percentage annotation for "Concurrency" specifies a percentage value
```

Comment:
You'll need to indent all the comments, otherwise it breaks the layout. It also might be nice to format them as notes, i.e.

```suggestion
            autoscaling.knative.dev/target-utilization-percentage: "70" 
    ```
    !!! note
        The `autoscaling.knative.dev/target-utilization-percentage` annotation for "Concurrency" specifies a percentage value
```

This should be done for all the updates

Reply:
Hello @psschwei 
Sure, I added the note section and content to next line by splitting it by br tag. enter was not working to move the content to next line same line as viewed in preview section.